schema.ts

```diff
@@ -427,6 +427,9 @@ export interface ClassMethod extends FunctionLike {
  * argument, so consumers of this interface should assume that the return type
  * is the single argument subclassed by this declaration.
  * 
+ * A mixin should only have a superclass if it composes another mixin, and the
```

Comment:
Im not sure I understand.

Given:
```js
export const MixinA = base => class extends MixinB(base){}
```

Here the `MixinB` would be `"superclass"`?

What in the case of:
```js
export const MixinA = base => class extends MixinC(MixinB(base)){}
```
?

Reply:
This was updated to say "a mixins should _not_ have a superclass". In your examples the other mixins would be included in the mixins field of MixinA.